### PR TITLE
950 deprecate derive vars suppqual

### PR DESCRIPTION
**Pull request by millerg23**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/) 
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiral/articles/unit_test_guidance.html#writing-unit-tests-in-admiral-)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiral/articles/programming_strategy.html#deprecation-1)?
- [x] Update to all relevant roxygen headers and examples 
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md`
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging. 
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done!  Much love to your accomplishment!

### CODE REVIEW

**Comment by millerg23** [comment]
I have an issue with `pkgdown::build_site()`, it is failing. I tried running in `devel` branch and get same ERRORs? Can anyone run this, is it just my local area that has the issue?
I also had trouble running `devtools::document()`, and had to update packages `pkgdown` and `roxygen2` to be same as versions in `main` branch. Any help to resolve this would be appreciated.

File: NEWS.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ also addresses the issue in the downstream functions `derive_vars_dt()` and `der
 
 - `derive_var_lstalvdt()` has been deprecated in favor of `derive_var_extreme_dt()` (#753).
 - `derive_vars_disposition_reason()` now is updated such that the default is populating `DCSREASP` only when `DSDECOD` is equal to `'OTHER'`, which is consistent with ADaMIG_v1.3 (#886).
+- `derive_vars_suppqual` has been removed from {admiral} as adding supplementary qualifiers is now done in another package called {metatools} in a function called `combine_supp()` (#950)
```
Comment (bms63) [inline NEWS.md:58]: can we link to the metatools packages?
```suggestion
- `derive_vars_suppqual` has been removed from {admiral} as adding supplementary qualifiers is now done in another package called [{metatools}](https://github.com/pharmaverse/metatools) in a function called `combine_supp()`  and is available on CRAN (#950)
```

File: R/derive_vars_suppqual.R [hunk 2]
```diff
@@ -1,5 +1,10 @@
 #' Join Supplementary Qualifier Variables into the Parent SDTM Domain
 #'
+#'#' @description
```
Comment (bundfussr) [inline R/derive_vars_suppqual.R:3]: Typo: extra `#'`

File: NEWS.md (hunk 1 above)
Comment (bundfussr) [inline NEWS.md:58]: Could you add brackets to the function name, i.e., `derive_vars_suppqual()`? Then it links to the documentation.

**Review by bundfussr — CHANGES_REQUESTED** [review]
- Please remove "The only exception is [derive_vars_suppqual()](https://r.roche.com/s/b44ca031f6d50e8019740/files/test_admiral/admiral/docs/reference/derive_vars_suppqual.html) which takes it from its argument dataset_suppqual, where the labels are stored." from the "Metadata" section in the programming strategy.
- Please remove the `derive_vars_suppqual()` call in `ad_adcm.R`.
- I think we can remove `assert_is_supp_domain()` from `assertions.R`.

**Comment by bundfussr** [comment]
Running `build_site()` worked for me after updating `tidyr` package to version 1.0.2.

File: NEWS.md (hunk 1 above)
Comment (millerg23) [reply NEWS.md]: Done, thanks, good suggestion.

Comment (millerg23) [reply NEWS.md]: Done, thanks.

File: R/derive_vars_suppqual.R (hunk 2 above)
Comment (millerg23) [reply R/derive_vars_suppqual.R]: Fixed.

**Comment by millerg23** [comment]
@bms63 and @bundfussr thanks very much for QC comments, think I have addessed them all. Still having issues building web site. I have `tidyr 1.0.2` installed, and I ran `renv::restore`. Will have another look and see what issue is.

**Review by bundfussr — APPROVED** [review]

**Review by bms63 — APPROVED** [review]

**Comment by bms63** [comment]
Hey Gordon.  Sorry!  Lots of merge conflicts we have been busy the last two days!

**Comment by millerg23** [comment]
Everything running clean  now, will see what checks bring up. May need to run styler if there are ERRORs there.

**Comment by millerg23** [comment]
@bms63 I am hopeful everything resolved now. Just waiting on a few checks to run.

**Review by bms63 — APPROVED** [review]
Approving again to fire check_templates

**Pull request merged**